packages/esm-patient-medications-app/src/add-drug-order/drug-search/drug-search.component.tsx

```diff
@@ -14,7 +14,8 @@ export default function DrugSearch({ openOrderForm }: DrugSearchProps) {
   const { t } = useTranslation();
   const isTablet = useLayoutType() === 'tablet';
   const [searchTerm, setSearchTerm] = useState('');
-  const debouncedSearchTerm = useDebounce(searchTerm);
+  const { debounceTime } = useConfig();
+  const debouncedSearchTerm = useDebounce(searchTerm, debounceTime);
```

Comment:
Cool. So the next thing to do is annotate the `useConfig` import with the `ConfigObject` from the medications app schema like so:

```tsx
import { type ConfigObject } from '../../config-schema';

// ... more code omitted for brevity
  const { debounceDelayInMs } = useConfig<ConfigObject>();
  const debouncedSearchTerm = useDebounce(searchTerm, debounceDelayInMs ? debounceDelayInMs : 300);
```